superx/routes/signup.py

```diff
@@ -0,0 +1,49 @@
+from app import app
```

Comment:
You'll want to rebase the code, this code was already changed in another PR

Reply:
ill look into what rebaseing means and how to do it